templates/terraform/bootstrap/secrets/main.tf

```diff
@@ -26,7 +26,7 @@ resource "aws_iam_access_key" "ci_user" {
 module "ci_user_keys" {
   source  = "../../modules/secret"
 
-  name_prefix    = "ci-user-aws-keys"
+  name    = "ci-user-aws-keys<% index .Params `randomSeed` %>"
```

Comment:
Just a general question... is randomSeed used to create a unique namespace that modules share for secrets, or does it actually get as a seed in random number generation by the modules?

Reply:
It just gets used as a suffix for the secret name, as secrets have an annoying behaviour where they are not fully deleted, so if you delete a secret and then try to use the same name again it fails. Not a big deal in the real world but it's annoying for testing.